src/Infrastructure/Connector/Action/Media/GetMedia.php

```diff
@@ -44,7 +44,10 @@ public function parseContent(?string $content): ?Shopware6Media
 
         $data = json_decode($content, true, 512, JSON_THROW_ON_ERROR);
 
-        return new Shopware6Media($data['data']['id'], $data['data']['attributes']['fileName']);
+        return new Shopware6Media(
+            $data['data']['id'],
+            $data['data']['attributes']['fileName'] ?: null
```

Comment:
are you sure `$data['data']['attributes']['fileName']` always exist?

`$data['data']['attributes']['fileName'] ?? null` would be safer but I'm not sure what's the purpose of this change ;)

Reply:
nice catch thank You